vendor/brave-ios/Ledger/Generated/NativeLedgerClientBridge.h

```diff
@@ -92,5 +92,9 @@
 - (void)deleteUnblindedTokensForPromotion:(const std::string&)promotion_id callback:(ledger::ResultCallback)callback;
 - (void)getTransactionReport:(const ledger::ActivityMonth)month year:(const uint32_t)year callback:(ledger::GetTransactionReportCallback)callback;
 - (void)getContributionReport:(const ledger::ActivityMonth)month year:(const uint32_t)year callback:(ledger::GetContributionReportCallback)callback;
+- (void)getNotCompletedContributions:(ledger::GetNotCompletedContributionsCallback)callback;
+- (void)getContributionInfo:(const std::string&)contribution_id callback:(ledger::GetContributionInfoCallback)callback;
+- (void)updateContributionInfoStepAndCount:(const std::string&)contribution_id step:(const ledger::ContributionStep)step retry_count:(const int32_t)retry_count callback:(ledger::ResultCallback)callback;
+- (void)updateContributionInfoContributedAmount:(const std::string&)contribution_id publisher_key:(const std::string&)publisher_key callback:(ledger::ResultCallback)callback;
```

Comment:
I'd say we should stick with Obj-C naming convention here (i.e. `retry_count` → `retryCount`),  _however_, if these methods are just going to be deleted with DB migration at a later date its fine to keep them how they are.

Reply:
yup all of it will be removed with db migration